src/vs/workbench/contrib/notebook/browser/viewModel/notebookOutlineEntryFactory.ts

```diff
@@ -73,9 +73,9 @@ export class NotebookOutlineEntryFactory {
 				// So symbols need to be precached before this function is called to get the full list.
 				if (cachedEntries) {
 					cachedEntries.forEach((cached) => {
-						entries.push(new OutlineEntry(index++, cached.level, cell, cached.name, false, false, cached.position, cached.kind));
+						const position = { ...cached.range, endLineNumber: cached.range.startLineNumber, endColumn: cached.range.startColumn };
```

Comment:
`Range.getStartPosition(cached.range)` is more readable.

Reply:
The selection options for openEditor doesn't match up with the type provided by getPosition, although, very strangly, the type system doesn't catch that.

```
export interface ITextEditorSelection {
	readonly startLineNumber: number;
	readonly startColumn: number;
	readonly endLineNumber?: number;
	readonly endColumn?: number;
}
```

vs 

```
export class Position {
	readonly lineNumber: number
	readonly column: number;
```